### DISCUSSION TITLE
Verify txt record in vercel

### DISCUSSION THREAD

**Discussion by dimasandriano** [is-a-dev/register#6144] [Q&A] [answered]
How to add txt record? Because when i add my domain on vercel, Vercel requests verification by adding a txt record.

**Comment by wdhdev** [comment, marked as answer]
We don't support Vercel unfortunately.

**Reply by dimasandriano** [reply]
Does that mean it only supports github pages? Or can it be used on netlify or railway?

**Reply by wdhdev** [reply]
It can be used on pretty much anything except for Vercel or Netlify.

**Reply by dimasandriano** [reply]
Im sorry sir, why my subdomain dimasandriano.is-a.dev When accessed it is redirected to vercel. Even though I deleted deploy on vercel. Please help sir

**Reply by wdhdev** [reply]
Your website seems to be reachable on my end. Try clearing your browser's cache.

**Reply by dimasandriano** [reply]
Thank u sir
